composer/loggers/mosaicml_logger.py

```diff
@@ -96,6 +102,90 @@ def log_hyperparameters(self, hyperparameters: Dict[str, Any]):
     def log_metrics(self, metrics: Dict[str, Any], step: Optional[int] = None) -> None:
         self._log_metadata(metrics)
 
+    def log_analytics(self,) -> None:
+        if self.analytics_data is None:
+            return
+
+        # Fetch / cast metrics that we want to log from self.analytics_data
+        autoresume: bool = self.analytics_data['autoresume']
```

Comment:
if we want to extend this down the line, how easy is it to add a new metric? do we just have to add a new field in the `analytics_data` dict?

Reply:
yup! after adding it to the `analytics_data`, you can log it to metadata however you like!

once it's being logged in the metadata, we can easily pull it out and add it into our [existing table for metrics](https://adb-2548836972759138.18.azuredatabricks.net/explore/data/users/anna_pfohl/genai_runs?o=2548836972759138) of run metrics. 

this is all done from a [notebook](https://github.com/mosaicml/mcloud/pull/3706), pulling out a new value would only be a few lines. Depending on where the metric is pulled from (i.e if it's also in `parameters`), we can backfill these metrics pretty easily as well